commit 9c5efd1afd49a57c4d17fc9245995fc4e1e6f653
Author: Michal Privoznik <mprivozn@redhat.com>
Date:   Tue Jun 16 01:42:04 2015 +0300

    qemuMigrationBeginPhase: Fix function header indentation
    
    This function is returning a string (domain XML). Since d3ce7363
    when it was first introduced, it was indented incorrectly:
    
    static char
    *qemuMigrationBeginPhase(..)
    
    Signed-off-by: Michal Privoznik <mprivozn@redhat.com>

diff --git a/src/qemu/qemu_migration.c b/src/qemu/qemu_migration.c
index b623fbc..ba6f6e5 100644
--- a/src/qemu/qemu_migration.c
+++ b/src/qemu/qemu_migration.c
@@ -2699,14 +2699,14 @@ qemuMigrationCleanup(virDomainObjPtr vm,
 
 
 /* The caller is supposed to lock the vm and start a migration job. */
-static char
-*qemuMigrationBeginPhase(virQEMUDriverPtr driver,
-                         virDomainObjPtr vm,
-                         const char *xmlin,
-                         const char *dname,
-                         char **cookieout,
-                         int *cookieoutlen,
-                         unsigned long flags)
+static char *
+qemuMigrationBeginPhase(virQEMUDriverPtr driver,
+                        virDomainObjPtr vm,
+                        const char *xmlin,
+                        const char *dname,
+                        char **cookieout,
+                        int *cookieoutlen,
+                        unsigned long flags)
 {
     char *rv = NULL;
     qemuMigrationCookiePtr mig = NULL;
